packages/blockchain-wallet-v4-frontend/src/data/analytics/types/coinView.ts

```diff
@@ -0,0 +1,12 @@
+export enum Events {
+  COINVIEW_EARN_REWARDS_BUTTON_CLICKED = 'Coin View Earn Rewards Clicked'
+}
+
+type CoinViewEarnRewardsButtonClick = {
+  key: Events.COINVIEW_EARN_REWARDS_BUTTON_CLICKED
+  properties: {
+    coin: string
```

Comment:
Should this be currency?

Reply:
And what about device and platform?

Reply:
Device and platform added for the tracking of the event. Please review.